docs/Compile.rst

```diff
@@ -108,9 +105,9 @@ extra options.
 You can also use a cmake-friendly IDE like KDevelop 4 or the cmake-gui
 program.
 
-Fixing the libstdc++ version bug
-================================
 
+The libstdc++ version bug
```

Comment:
"Fixing the libstdc++ version issue" might be a better title (ABI-related errors like this are not bugs).